### PR TITLE
Add test to ensure that we can render stories from SWR call.

### PR DESCRIPTION
**Pull request by diogob**

### CODE REVIEW

File: ui/__tests__/index.test.tsx [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
 
     expect(screen.getByText(/Stories/)).toBeInTheDocument()
   })
+
+  it('renders story titles', () => {
+    render(<TodosPage />)
+
+    expect(
+      screen.getAllByText(/write test files/).length
+    ).toBeGreaterThanOrEqual(1)
```
Comment (gustavoguichard) [inline ui/__tests__/index.test.tsx:32]: @diogob , any special reason for asserting with such imprecise method?

**Review by gustavoguichard — APPROVED** [review]

File: ui/__tests__/index.test.tsx (hunk 1 above)
Comment (diogob) [reply ui/__tests__/index.test.tsx:32]: this text shows more than once, once in the story title and once in the description.
I'll use a matcher to get only the title instance

Comment (gustavoguichard) [reply ui/__tests__/index.test.tsx:32]: Maybe we could change the title of the stories from `span` to some heading like `h4` for instance. Then you'll be able to select it with `screen.getByRole('heading', { name: /write test files/i })`

Comment (diogob) [reply ui/__tests__/index.test.tsx]: I have tightened the expectation by requiring 2 ocurrences. Later we can improve on the matcher if we need something more precise

**Pull request merged**